ONSdigital/design-system
# Update prepend-file to v2

### What is the context of this PR?
Updates prepend-file to v2.

### How to review
- In `prepare-templates-for-zip.js` change `process.env.RELEASE_VERSION` to something 
- Run `yarn cdn-bundle` and check it runs without error
- See that in ./templates/layout/_template.njk that the first line includes setting the release version to what you replaced `process.env.RELEASE_VERSION` with
